src/krb5_plugin/sssd_krb5_locator_plugin.c

```diff
@@ -414,6 +415,7 @@ krb5_error_code sssd_krb5_locator_init(krb5_context context,
         ctx->disabled = true;
         PLUGIN_DEBUG("SSSD KRB5 locator plugin is disabled.\n");
     }
+    ctx->kpasswdinfo_used = false;
```

Comment:
This is not harmful, but not needed as the structure is allocated with calloc

Reply:
Yes, but I hope it will make the code more readable if the default is set explicitly.